docs/kb/customizing-cpuset.rst

```diff
@@ -0,0 +1,105 @@
+
+==========================
+Customizing CPUSET
+==========================
+
+CPUSET in the ``/etc/scylla.d/cpuset.conf`` file is automatically configured by running the ``scylla_setup`` 
+(or ``scylla_cpuset_setup``) script. If you want to customize CPUSET, you must modify both the ``cpuset.conf`` 
+and ``perftune.yaml`` files.
+
+
+Note that:
+
+* The ``scylla-server`` service will try to generate ``/etc/scylla.d/perftune.yaml`` *only if* the file doesn't exist.
+* The ``scylla-server`` service will successfully generate ``/etc/scylla.d/perftune.yaml`` *only if* the value set in ``cpuset.conf`` 
+  corresponds to one of the default tuning modes: ``mq``, ``sq``, or ``sq_split``. If the CPUSET value doesn't match any of these modes, 
+  the ``scylla-server`` service will refuse to start, throwing a ``tune mode not found`` error.
+
+
+Examples
+----------
+Here are a few examples of possible configurations. In the examples, ``eth5`` is used as a NIC name and ``/var/lib/scylla`` as a data directory.
+
+Example 1
+^^^^^^^^^^
+
+* 16 CPUs system.
+* You want to run Scylla on CPUs 3, 4, 5, and have IRQs handled on the same CPUs, while allowing other apps on the same machine to benefit 
+  from XFS/RFS/RPS from ``eth5``.
+
+**cpuset.conf:**
+
+.. code-block:: none
+
+    CPUSET="--cpuset 3,4,5 "
+
+**perftune.yaml:**
+
+.. code-block:: none
+
+    cpu_mask: '0xffff'
+    dir:
+    - /var/lib/scylla
+    irq_cpu_mask: '0x38'
+    nic:
+    - eth5
+    tune:
+    - net
+    - disks
```

Comment:
Interestingly the example corresponds to the modern perftune.py unlike lines I commented above.